### ISSUE TITLE
Theme JSON Using URL Containing Localhost

### ISSUE THREAD

**Issue by kawigreen14r** [structurizr/structurizr#119]
Hi,  Is an url with localhost allowed for using a custom theme file in the Lite version?  Or are there other rules I should be aware of?

The published themes from Structurizr work fine.  To test custom theme's, I need to use a local JSON (HTTP-Server) server on my workstation, as setting up a static file server in my companies environment is both time consuming and expensive.  Here is the url I use that correctly brings up the JSON file when put into the address bar, along with the urls for the images referenced in the json.  And just to rule out issues with the JSON file itself, I copied the content from the AWS version on your site and received the same error.

There are no console messages reported

Here is what the url looks like:
theme http://localhost:8888/public/aws/theme.json

The pop-up error message:
localhost:8080 says

Could not load theme from http://localhost:8888/public/aws/theme.json; error 0(error)

**Comment by simonbrowndotje** [comment]
It should work ... themes are loaded client-side, so perhaps take a look at the network tab in the developer console and see if you can spot the request.

**Comment by kawigreen14r** [comment]
Thanks for the tip, that found the problem.  Looks to be the "Access-Control-Allow-Origin" header needs to be set, which you referenced in your documentation.  I made an assumption though that since Lite runs locally with http and not https, that I didn't need to set that in my "fake" static server instance.

Here is the message I got in the Console using Inspect:

DevTools failed to load source map: Could not load content for http://localhost:8080/static/js/backbone-min.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
diagrams#CTCORE-Container:1 Access to XMLHttpRequest at 'http://localhost:8888/public/aws/theme.json' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
structurizr-diagram-2781.js:7524 Could not load theme from http://localhost:8888/public/aws/theme.json; error 0 (error)

I'll work on getting that header implemented.  Thanks again for responding so quickly, I'll close this issue
